llvm-spirv/lib/SPIRV/SPIRVInternal.h

```diff
@@ -373,6 +373,7 @@ const static char ConvertHandleToImageINTEL[] = "ConvertHandleToImageINTEL";
 const static char ConvertHandleToSamplerINTEL[] = "ConvertHandleToSamplerINTEL";
 const static char ConvertHandleToSampledImageINTEL[] =
     "ConvertHandleToSampledImageINTEL";
+const static char EntrypointPrefix[] = "__spirv_entry_";
```

Comment:
Very minor Nit: Can you please add this three lines earlier to align with upstream translator?

Thanks